.changes/unreleased/tinymce-TINY-10560-2024-02-15.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Adding newline after using `selection.setContent` with a `<pre>` caused a crash
```

Since this affects more than PRE we should expand it.

```suggestion
body: Adding newline after using `selection.setContent` insert a block element would throw an unhandled excception.
```